draft-ietf-quic-recovery.md

```diff
@@ -990,17 +989,20 @@ congestion without depending on PTO expiration.
 
 ### Establishing Persistent Congestion
 
-A sender establishes persistent congestion on receiving an acknowledgement if at
-least two ack-eliciting packets are declared lost, and:
+A sender establishes persistent congestion after the receipt of an
+acknowledgement if at least two ack-eliciting packets are declared lost, and:
 
-* all packets, across all packet number spaces, sent between these two send
-  times are declared lost;
+* all packets, across all packet number spaces, sent between the send times of
+  these two packets are declared lost;
 
 * the duration between the send times of these two packets exceeds the
   persistent congestion duration ({{pc-duration}}); and
 
 * a prior RTT sample existed when both packets were sent.
 
+The two packets to consider need to be ack-eliciting, because otherwise a sender
+cannot expect to receive an acknowledgement after a specific time.
```

Comment:
```suggestion
These two packets MUST be ack-eliciting, since a receiver is required to
acknowledge only ack-eliciting packets within its maximum ack delay; see Section
13.2 of {{QUIC-TRANSPORT}}.
```